packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -168,6 +168,17 @@ export function getInternalReactConstants(
     NoPriority: 90,
   };
 
+  if (gte(version, '17.0.2')) {
```

Comment:
@bvaughn How am I supposed to do this?

Reply:
Updated to my best guess based on the other checks:

https://github.com/facebook/react/blob/8b0532a015626d9cde960e1162b176e9e037cc77/packages/react-devtools-shared/src/backend/renderer.js#L171-L180